src/components/views/auth/PasswordLogin.tsx

```diff
@@ -484,7 +484,7 @@ export default class PasswordLogin extends React.PureComponent<IProps, IState> {
                     {forgotPasswordJsx}
                     { !this.props.busy && <input className="mx_Login_submit"
                         type="submit"
-                        value={_t('Sign in')}
+                        value={('Log in')}
```

Comment:
This breaks internationalisation in element.  I'd suggest updating the value in src\i18n\strings\en_EN.json